senseobjdef/constraint.go

```diff
@@ -138,7 +142,12 @@ func (constraint *Constraint) Evaluate(data json.RawMessage) (bool, error) {
 	case string:
 		return constraint.operator.evalString(value.(string), constraint.value)
 	default:
-		return false, errors.Errorf("value type<%T> not supported", value)
+		switch reflect.TypeOf(value).Kind() {
+		case reflect.Slice:
+			return constraint.operator.evalArray(value, constraint.value)
+		default:
+			return false, errors.Errorf("value type<%T> not supported", value)
+		}
```

Comment:
Do we expect more of the slice type here?

It might be overkill to use reflection instead of a fourth `case []string, []float64, []int` or whatever is appropriate.

Reply:
You're right, that would be better!